docs/time/index.rst

```diff
@@ -276,13 +276,18 @@ tdb    Barycentric Dynamical Time  (TDB)
 tt     Terrestrial Time            (TT)
 ut1    Universal Time              (UT1)
 utc    Coordinated Universal Time  (UTC)
+local  Local Time Scale            (LOCAL)
 ====== =================================
-
+  
 .. [#] Wikipedia `time standard <https://en.wikipedia.org/wiki/Time_standard>`_ article
 .. [#] SOFA Time Scale and Calendar Tools
        `(PDF) <http://www.iausofa.org/sofa_ts_c.pdf>`_
 .. [#] `<http://www.ucolick.org/~sla/leapsecs/timescales.html>`_
 
+.. note:: ``local`` TimeScale is a scale that cannot be converted to/from any
+   other scale and to which only TimeDelta with ``None`` type scale could be
+   added or subtracted.
+
 The system of transformation between supported time scales is shown in the
```

Comment:
I think maybe here note again that ``local`` is excluded; then we do not have to update the figure. E.g.,
```
The system of transformations between the standard time scales (i.e., all but ``local``)
is shown in the ...
```